domains/emilia.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "Hazaelake"
+    },
+    "record": {
+        "RECORDTYPE": "RECORDCONTENT"
```

Comment:
Invalid record type and record content. See docs for actual proper guide: https://docs.is-a.dev